Source/WTF/wtf/DateMath.cpp

```diff
@@ -470,6 +470,19 @@ static bool parseInt(const char* string, char** stopPosition, int base, int* res
     return true;
 }
 
+static bool parseInt(std::span<const LChar>& string, int base, int* result)
+{
+    char* stopPosition;
+    long longResult = strtol(reinterpret_cast<const char*>(string.data()), &stopPosition, base);
+    // Avoid the use of errno as it is not available on Windows CE
+    if (string.data() == reinterpret_cast<const LChar*>(stopPosition) || longResult <= std::numeric_limits<int>::min() || longResult >= std::numeric_limits<int>::max())
+        return false;
+    string = string.subspan(reinterpret_cast<const LChar*>(stopPosition) - string.data());
+    *result = static_cast<int>(longResult);
```

Comment:
This `static_cast<int>` is not needed, has no effect since assigning already chops `long` to `int`.

Reply:
Fixed.